python/lsst/daf/butler/registry/_dbAuth.py

```diff
@@ -79,6 +83,8 @@ def __init__(self, path=None, envVar=None, authList=None):
                 "{:o}".format(secretPath, mode))
 
         try:
+            if not os.path.isfile(secretPath):
+                raise DbAuthNotFoundError(f"DbAuth file not found: {secretPath}")
```

Comment:
This should be removed and the proper exception thrown at (new) line 78.

Reply:
Thanks! I guess I was blind.